### PR TITLE
Created the knowing index 

### PR DESCRIPTION
**Pull request by fouziiah**
### Changes
- Created routes, controller and view for the Knowing_Category model
- Create a seed data for the student knowing 

### Link to Issue 
- https://app.asana.com/0/1206510424537042/1206645498622404/f

### Comments, Notes, or  Questions
- None

### Related Screenshots 
Screenshot 2024-04-17 at 11 41 29 AM

### Before you submit for review, Did you...

- [x] Added relevant tests (if applicable)
- [x] Linked Asana ticket to PR
- [x] Added comments, notes, or questions for reviewer
- [x] Added relevant screenshots
- [x] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-load(Rails.root.join('db/seeds/classroom_setup.rb')) if Rails.env.development?
+# load(Rails.root.join('db/seeds/classroom_setup.rb')) if Rails.env.development?
```
Comment (edithemmings) [inline db/seeds.rb:3]: What was the reason for commenting this out? It looks like we still need it

**Review by edithemmings — COMMENTED** [review]
This looks great! Super clean implementation and the UI looks great. I left one question about the seeds

File: db/seeds.rb (hunk 1 above)
Comment (fouziiah) [reply db/seeds.rb:3]: oops, I didn't realized i did.

**Review by camckin10 — APPROVED** [review]
✅ Nice work!

**Pull request merged**